crates/contract/src/primitives/participants.rs

```diff
@@ -204,12 +204,16 @@ impl Participants {
         }
     }
 
+    /// Returns the set of NodeIds corresponding to the participants.
+    /// Note that the account_public_key field in NodeId is None.
+    /// This is because NodeId is used in contexts where account_public_key is not needed (only TLS key is needed).  
```

Comment:
```suggestion
    /// Note that the `account_public_key` field in [`NodeId`] is `None`.
    /// This is because [`NodeId`] is used in contexts where `account_public_key` is not needed (only TLS key is needed).  
```